pkg/context/apikey.go

```diff
@@ -106,6 +107,25 @@ func (c APIContext) ChangeContext(key string) error {
 	return nil
 }
 
+func (c APIContext) RemoveContext(key string) error {
+	if key == "" {
+		return ErrorInvalidKey
+	}
+
+	d, err := c.DBOpen()
+	if err != nil {
+		return err
+	}
+	defer d.Close()
+
+	// Delete context key
+	if err := d.Delete(c.ContextKey); err != nil {
```

Comment:
d.Delete should be called with the passed in key param instead of the context

d.Delete(key);

Reply:
yes I saw that and fixed it. Still fails the test for RemoveContext